exercises/D-logical-operators/exercise.js

```diff
@@ -11,14 +11,14 @@ let cssLevel = 4;
 
 // Finish the statement to check whether HTML, CSS knowledge are above 5
 // (hint: use the comparison operator from before)
-let htmlLevelAbove5;
-let cssLevelAbove5;
+let htmlLevelAbove5 = true;
+let cssLevelAbove5 = false;
 
 // Finish the next two statement
 // Use the previous variables and logical operators
 // Do not "hardcode" the answers
-let cssAndHtmlAbove5;
-let cssOrHtmlAbove5;
+let cssAndHtmlAbove5 = false;
```

Comment:
Exercise is telling You to use the comparison operator to prove that, for example, there are more students than mentors. For example:
let moreStudentsThanMentors = studentCount > mentorCount

Please check this link: https://www.w3schools.com/js/js_comparisons.asp